test/e2e/network_test.go

```diff
@@ -80,7 +80,6 @@ var _ = Describe("Podman network", func() {
 	)
 
 	It("podman network list", func() {
-		SkipIfRootless()
```

Comment:
Why are we dropping the rootless checks?

Reply:
He moved it up to the top.

Reply:
it got moved up into the pre functions right?